roles/proxy/templates/nginx_artemis.conf.j2

```diff
@@ -77,13 +77,6 @@ server {
 {% endif %}
 
 {% if proxy_athene_baseurl is not none %}
```

Comment:
Maybe we can re-write this as `{% if athene is defined %}` and drop the `proxy_athene_baseurl` from [main.yml](https://github.com/ls1intum/artemis-ansible-collection/blob/main/roles/proxy/defaults/main.yml) entirely?

I'm not confident enough in my understanding of ansible to do this myself, though. I would appreciate it if someone more knowledgeable could look into whether that's possible.

Reply:
Is the case distinction even necessary? We could just add it by default - If Athene is not enabled in the Artemis instance the path will just return an error right?

Reply:
That's true! I removed it now 👍 
I could imagine it was necessary before because otherwise, the `proxy_athene_baseurl` wouldn't have been defined - which might have caused problems. However, it's unnecessary now so that the check can be removed.